tools/bumper/cnaoRepoCommands.go

```diff
@@ -0,0 +1,168 @@
+package main
```

Comment:
In golang go sources fails are usually lowercase, let's move this to just `cnaorepocommands.go`

Reply:
I'll keep it like the other files in the bumper folder: cnao_repo_commands.go
Done